### ISSUE TITLE
Heading Hierarchy on Search Results Page

### ISSUE THREAD

**Issue by mtribone** [samvera/hyrax#1263]
### Descriptive summary

In Hyrax 2.0.0 alpha the heading hierarchy on the search results page could be better to improve the document outline and accessibility of the page. As it stands, the document hierarchy is heavy on `<h2>` and `<h3>` and the relationships aren't clear.

### Rationale

Headings used for structure and not formatting are used by users of assistive technology to navigate and gain understanding of a page. pages with poor hierarchy or strucure make it difficult for users to understand what to do, where to go, or understand the content of a page.

### Expected behavior

Properly nested heading hierarchy from `<h1>` through `<h6>`

### Actual behavior

Mostly `<h2>` and `<h3>` with poor hierarchy

### Steps to reproduce the behavior

1. Inspect the page and view source code
1. Locate headings and match to visual hierarchy on the page

### Related work

Epic rework. Breaking these up in Hyrax

https://github.com/psu-stewardship/scholarsphere/issues/582
https://github.com/psu-stewardship/scholarsphere/pull/614

**Comment by mtribone** [comment]
![screen shot 2017-06-22 at 9 08 21 am](https://user-images.githubusercontent.com/4163828/27435583-a21918a8-572a-11e7-8246-8f0fe57fed44.png)

**Comment by jcoyne** [comment]
This might be something to fix upstream in blacklight.  I notice there's no `<h1>` on this page: https://demo.projectblacklight.org/en?utf8=%E2%9C%93&search_field=all_fields&q=

**Comment by mtribone** [comment]
@jcoyne Yeah definitely upstream but wanted to get thoughts here with about 4 tickets that relate to one another.

**Comment by jcoyne** [comment]
@mtribone This would be a good fix to get into Blacklight before version 7 is released.

**Comment by adamjarling** [comment]
Here's an update for 2018; the following 2 issues look like they need to be fixed (in Blacklight guessing?)

- [ ] Missing `<h1>` on page
- [ ] The search title `<h4>` says it skips a level.  

<img width="1258" alt="headers-in-order-search-results" src="https://user-images.githubusercontent.com/3020266/45303106-a5109e00-b4da-11e8-8fbf-b0d353840fb1.png">

**Comment by rjkati** [comment]
In Hyrax 3.0.0. rc3:

- [ ]  `h1` is present 
- [ ] the search title `<h4>` has skipped a level

![hyraxsearchpage2](https://user-images.githubusercontent.com/32644254/105759702-bf6a3b80-5f1e-11eb-98c7-da84cf75cb4e.PNG)

**Comment by jeremyf** [comment]
For testing this, I disable the page's stylesheet.  This allows me to look at the hierarchy.

```
H1 - Page Title
H2 - Search Constraints
H2 - Search Results
H4 - Result 2
H4 - Result 2
etc.
```

It looks like the `h4.search-result-title` should be `h3.search-result-title`.  I'm exploring how to make that change.

**Comment by jlhardes** [comment]
The heading hierarchy on the search results page is now properly nested. This is fixed.
